src/ssb_project_cli/ssb_project/app.py

```diff
@@ -282,7 +287,7 @@ def create(
         # Link does not work in jupyter labs følg https://statistics-norway.atlassian.net/wiki/spaces/DAPLA/pages/1917779969/Oppstart+personlig+GitHub-bruker+personlig+kode+og+integrere+Jupyter+med+GitHub#Opprette-personlig-aksesskode-i-GitHubinstruksjonene her for å skape en
     ),
 ) -> None:
-    """:sparkles: Skap et prosjekt lokalt og på Github (hvis ønsket).Følger kjent beste praksis i SSB. :sparkles:."""
+    """:sparkles: Skap et prosjekt lokalt og på Github (hvis ønsket). Følger kjent beste praksis i SSB."""
```

Comment:
Bruk `Opprett` i stedet for `Skap`. Hva menes med `hvis ønsket`? Jeg ser ikke noe flagg for å angi dette

Reply:
Jeg endrer det til opprett. Det finnes jo en flagg, `--github`.

Reply:
Da synes jeg at teksten bør bli slik: "Opprett et prosjekt lokalt, og på Github hvis flagget `--github` er satt. Prosjektet følger beste kodekvalitetspraksis i SSB."